bcryptjs/index.d.ts

```diff
@@ -16,21 +16,26 @@ export declare function setRandomFallback(random: (random: number) => number[]):
  * Synchronously generates a salt.
  * @param  rounds Number of rounds to use, defaults to 10 if omitted
  * @return Resulting salt
+ * @throws If a random fallback is required but not set
  */
 export declare function genSaltSync(rounds?: number): string;
 
 /**
  * Asynchronously generates a salt.
- * @param callback Callback receiving the error, if any, and the resulting salt
+ * @param  callback Callback receiving the error, if any, and the resulting salt
+ * @return Promise with resulting salt, if callback has been omitted
+ * @throws If callback is present but not a function
  */
-export declare function genSalt(callback: (err: Error, salt: string) => void): void;
+export declare function genSalt(callback?: (err: Error, salt: string) => void): Promise<string>;
```

Comment:
Thank you very much! This is my first definition and wasn't really sure, how to solve that. Thanks!